include/psa/crypto.h

```diff
@@ -415,7 +415,9 @@ void psa_reset_key_attributes(psa_key_attributes_t *attributes);
  * \retval #PSA_ERROR_BAD_STATE
  *         The library has not been previously initialized by psa_crypto_init().
  *         It is implementation-dependent whether a failure to initialize
- *         results in this error code.
+ *         results in this error code. Or,
+ *         this call was operating on a key slot and found the slot in
+ *         an invalid state for the operation.
```

Comment:
We don't need this change if we change `psa_key_slot_state_transition()` to return `PSA_ERROR_CORRUPTION_DETECTED`.